sql/sql_show.cc

```diff
@@ -1872,12 +1884,12 @@ class Fill_authinfo_list : public Do_THD_Impl {
                    ? nullptr
                    : thd->security_context()->priv_user().str) {}
 
-  virtual void operator()(THD *current_thd) override {
-    const auto current_sctx = current_thd->security_context();
+  virtual void operator()(THD *iteration_thd) override {
```

Comment:
@hermanlee It does not, but `current_thd` is a special global var in MySQL (even a macro in some flavors), so I did this change just to avoid potential magic behavior which would be hard to understand.
This change should have been done as part of the
FB8-53: Expose user certificate details to command line
https://jira.percona.com/browse/FB8-53
https://github.com/facebook/mysql-5.6/commit/eda838a6063afed06f273f486cf7900b1ac1e00f

Reply:
Ah, okay. We can fix the current_thd in a future rebase for that patch. I'll separate this change out into another diff so that it can be squashed together.